builtin/logical/cassandra/path_config_connection.go

```diff
@@ -119,6 +124,7 @@ func (b *backend) pathConnectionWrite(
 		TLS:             data.Get("tls").(bool),
 		InsecureTLS:     data.Get("insecure_tls").(bool),
 		ProtocolVersion: data.Get("protocol_version").(int),
+		ConnectTimeout:  data.Get("connect_timeout").(string),
```

Comment:
Once you've made the change up above you can simply do a `data.Get("connect_timeout").(int)`